app/javascript/imagesort.js

```diff
@@ -0,0 +1,26 @@
+import Sortable from 'sortablejs'
+console.log('sortable', Sortable)
+
+var el = document.getElementById('image-list');
+const artworkId = el.getAttribute('data-artwork-id')
+console.log(el)
+var sortable = Sortable.create(el, {
+    handle: '.sort-handle',
+    animation: 100,
+    filter: '.ignore-drag',
+    onEnd: async e => {
+        const imageId = e.item.getAttribute('data-image-id')
+        const url = `/artworks/${artworkId}/images/${imageId}/move_image?from_index=${e.oldIndex}&to_index=${e.newIndex}`
+        const response = await fetch(url, { 
+            method: 'PATCH',
+        })
+        if(response.status != 200) {
+            // just in case there is a server error
+            alert('Changing image order failed. Please refresh the page and try again.')
```

Comment:
I wanted to revert the drag in case of an error but couldn't find a way to do that and I don't see any scenario of this failing other than server being down or something so an alert should be good enough.